src/users/user.php

```diff
@@ -156,14 +156,23 @@ public function get()
 	{
 		$input = JFactory::getApplication()->input;
 		$id = $input->get('id', 0, 'int');
+		$xidentifier	= $input->server->get('HTTP_IDENTIFIER', '', 'String');
```

Comment:
The header needs to be `X-Identifier`. Please change that in the user update/save method also.

Reply:
@coolbung Updated Please check